pygfx/renderers/wgpu/engine/pipeline.py

```diff
@@ -679,7 +683,7 @@ def _check_render_info(self):
         assert all(isinstance(i, int) for i in indices)
         assert len(indices) == 3
 
-    def _compile_shader(self, pass_index, env):
+    def _compile_shader(self, pass_index, env, material=None):
         """Compile the templateds wgsl shader to a wgpu shader module."""
         return get_cached_shader_module(self.device, self.shader, {})
```

Comment:
now that the material can control the pickability, shouldn't this affect the result of the cache too?

Reply:
hmm this is the compute shader, so i guess it isn't so important.